package.json

```diff
@@ -19,7 +19,6 @@
   "author": "John Hiesey",
   "license": "MIT",
   "dependencies": {
-    "inherits": "^2.0.1",
-    "readable-stream": "^2.0.5"
+    "readable-stream": "^3.0.2"
```

Comment:
@jhiesey I think this could create _"a little mess"_ within videostream.js etc, I.e. `readable-stream` should ideally also get upgraded on `mediasource` & `mp4-stream` (?)

```bash
$ npm ls readable-stream
videostream@2.6.0
+-- mediasource@2.2.2
| `-- readable-stream@2.3.6
+-- mp4-stream@2.0.3
| `-- readable-stream@2.3.6
+-- range-slice-stream@2.0.0
| `-- readable-stream@3.0.3
`-- readable-stream@3.0.3
```

Reply:
I know
I'm trying to help all package involved in webtorrent to try and update readable-stream to v3

https://npm.anvaka.com/#/view/2d/webtorrent

Reply:
Thanks! :+1: